### PR TITLE
docs(general): Fix typos in documentation

### PR DESCRIPTION
**Pull request by LucasVera**
# Fix documentation typos

This PR addresses some typos I found while reading the [docs](https://serverlessguru.gitbook.io/sls-jest/). I focused only on minor typos and small grammar that don't modify the general idea.

As an improvement for the future, I think some more context is missing in the "Getting started" on what `sls-jest` is and what it isn't, instead of jumping straight into the commands to install..

### CODE REVIEW

File: doc/spies/eventbridge.md [hunk 1]
```diff
@@ -16,21 +16,21 @@ This helper creates a new spy for a given event bus. Parameters:
 
 Config:
 
-- `matcherDefaultTimeout`: The default timeout for event matchers, in milliseconds. Defaults to `10_000`. This is the maximum time a matcher will wait until it determines whether the assertion succeeds or fails. Also see the [recommendations](#recommendations) below about timeouts.
+- `matcherDefaultTimeout`: The default timeout for event matchers, in milliseconds. Defaults to `10000`. This is the maximum time a matcher will wait until it determines whether the assertion succeeds or fails. Also see the [recommendations](#recommendations) below about timeouts.
```
Comment (bboure) [inline doc/spies/eventbridge.md]: I actually made this one on purpose for readability.

**Review by bboure — APPROVED** [review]
Thank you!

Typos! This is all me 😆 

I try to re-read myself but I just read my typos 🤦‍♂️

File: doc/spies/eventbridge.md (hunk 1 above)
Comment (LucasVera) [reply doc/spies/eventbridge.md]: I think it may make sense, however I saw other numbers didn't have this convention so I thought it was a typo as well 😅

Comment (bboure) [reply doc/spies/eventbridge.md]: Yeah, we could make this consistent. 
I'm fine either way.

Comment (bboure) [reply doc/spies/eventbridge.md]: I probably thought that 3 `0` was readable but after 4 it's harder.

Comment (LucasVera) [reply doc/spies/eventbridge.md]: Reverted to use the `_` convention for big numbers

**Review by bboure — APPROVED** [review]

**Comment by bboure** [comment]
Thank you 🎉

**Review by bayoudhi — APPROVED** [review]
LGTM! Thank you @LucasVera for the fixes and improvements.

**Pull request merged**